hello-world-python/README.md

```diff
@@ -0,0 +1,110 @@
+   your enclave application.
+3. Simulation Mode - This does not require you to have an Intel SGX CPU. This can be used to test your conclave 
+   applications when in development mode. This mode requires a linux env. This can be run inside a Docker container 
+   on Mac. Enclave code is compiled to native binary and loaded inside a separate JVM inside your host JVM.
+4. Mock Mode - Enclave code can be directly run inside the host VM. This is very fast, and hence should ideally be used
+   for day to day rapid development and testing.
+
+To read more about modes go to the [docs](https://docs.conclave.net/enclave-modes.html).
+
+# Configurations to be added to a hello world conclave sample
+
+If you have not yet built any conclave application, take a look at this 
+[page](https://docs.conclave.net/writing-hello-world.html) which talks about how to run a simple hello world
+conclave application.
+
+To build the application using conclave init tool take a look [here](https://docs.conclave.net/conclave-init.html)
+
+Python is not enabled by default. 
+In order to enable support add the following line to your enclave build.gradle.
+
+      conclave {
+      productID = 1
+      ...
+      supportLanguages = "python"
+      }
+
+Adding this line pulls all the required python dependencies.
+
+Let's define our python function which we want to execute inside the enclave in the ReverseEnclave.class
+
+      private static final String pythonCode = "def reverse(input):\n"
+      + " return input[::-1]";
+
+Let us also define the Bindings and the Context 
+
+      private final Context context;
+      private final Value bindings;
+
+Let us now define the bindings in the constructor
+
+      public ReverseEnclave() {
+      context = Context.create("python");
+      bindings = context.getBindings("python");
+      context.eval("python", pythonCode);
+      }
+
+This sets up the bindings to Java which allow us to access the functions and variables that are defined within the 
+Python code.
+
+Below code snippet shows you how we can execute the function binded by use before.
+
+      private String reverse(String input) {
+        Value result = bindings.getMember("reverse").execute(input);
+        return result.asString();
+    }
+
+Now we are done with adding the configurations and now let's see hwo to run this sample in mock mode.
+
+# Steps to run the Hello World python sample in mock mode
+
+For this sample, we will run our sample in mock mode.
+There is development work going on to make it work in the simulation and other modes.
+Before running this sample in mock mode make sure to perform below steps.
+
+1. Download [graalvm-ce-java11](https://github.com/graalvm/graalvm-ce-builds/releases). 
+2. Set the environment variable JAVA_HOME to point to the GraalVM that was previously downloaded. 
+   For instance, export JAVA_HOME=/usr/lib/jvm/graalvm-ce-java11-21.1.0).
+3. Update the environment variable PATH by running export PATH=JAVA_HOME/bin:$PATH.
```

Comment:
The dollar symbol is missing from the environment variable JAVA_HOME. It should be "PATH=$JAVA_HOME/bin:$PATH"

Reply:
yes, correct. done.